advcpmv-0.8-8.32.patch

```diff
@@ -204,11 +204,13 @@ diff -aur coreutils-8.32/src/copy.c coreutils-8.32-patched-nc/src/copy.c
 +            format_time(f_ttime, sec_remaining, true);
 +            format_time(f_ftime, sec_fremaining, true);
 +
-+            sprintf ( s_progress->cProgressField[1],
++            int fs_len;
++            fs_len = sprintf ( s_progress->cProgressField[1],
```

Comment:
i'd use `snprintf`, just to be safe. also worth checking for failures `if (fs_len < 0)`.